cdap-standalone/bin/cdap.sh

```diff
@@ -109,7 +109,7 @@ fi
 NODE_INSTALL_STATUS=$(program_is_installed node)
 if [ "x$NODE_INSTALL_STATUS" == "x1" ]; then
   die "Node.js is not installed
-Please install Node.js - we recommend any version of Node.js greater than v0.10.0."
+Please install Node.js - we recommend any version of Node.js from v0.10.36 through v0.12.*"
```

Comment:
we do a lot more testing for 0.10.36 - 0.12.0. However it is working on 4.x and 5.x, I just have not done an in depth tests for it to comfortably say we support it 100%.